index.d.ts

```diff
@@ -21,7 +21,9 @@ declare global {
     readonly timer?: TimerAPI
     readonly viewport: ViewportAPI
 
-    readonly currentUser: CurrentUser | null
+    readonly currentUser: User | null
```

Comment:
Good point. I still think changing the name is better, however, because the `currentUser` and `activeUser.user` types should be the same, and `User` is the most generic name for it. @brsbl  is this an issue if this breaks existing plugins?

Reply:
@brsbl  said it is not a big deal because there was never an official release note about it. Here is direct quote (can't link slack due to it being a DM): 

> i think this is ok because we never announced figma.currentUser -- we just silently released it for the widget beta, so i doubt anyone is relying on it (edited)

Reply:
nice! looks good

Reply:
Changing TypeScript types doesn't actually ever break plugins. All the type information is removed by the TypeScript->JS compiler, so any plugin code will continue to work. It can affect errors that users see in their development environment, requiring to change code in order to address those errors, but we've always felt ok with that

Reply:
This is a bummer. I was using it :(. Can someone catch me up on why this shouldn't exist?

Reply:
Hi @volanStudio these changes are still in the works. I am actually closing this PR because it is outdated now.